plugins/koa/koaintegrationplugin.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+const compose = require('koa-compose');
+const Router = require('koa-router');
+const router = new Router();
+const reconfig = require('@brillout/reconfig');
+
+module.exports = {
+    name: 'koaIntegrationPluginFile',
+    multiple: false,
+    routes: routes(),
+};
+
+function routes() {
+	return compose(
+	[
+		router.routes(),
+		router.allowedMethods()
+	]
+	)
+}
+
+router.get('*', async(ctx, next) => {
+
+    const {body, headers} = await getResponse(ctx);
+
+    if (body === null) {
+        ctx.status = 404;
+        return;
+    }
```

Comment:
Why return a 404? Wouldn't it undercut koa routes the user may have added?

Reply:
You're right